### ISSUE TITLE
closed account

### ISSUE THREAD

**Issue by rolly2002** [Eqonomize/Eqonomize#46]
is it possible to re-open a closed account? 
tank you.

**Comment by hanna-kn** [comment]
Yes. Select a period with non-zero change or total for the closed account in the account view. The account will reappear in the list and it can be reopened in the same way that it was closed.
